sdk/dnsresolver/azure-resourcemanager-dnsresolver/tsp-location.yaml

```diff
@@ -1,4 +1,4 @@
 directory: specification/dnsresolver/resource-manager/Microsoft.Network/DnsResolver
-commit: 3ec630fc62ebc03e9adf238cb2fb0b538d93ead4
+commit: efa49a123da7ce3ffe093a13832258305f529711
```

Comment:
Just to confirm my understanding: when updating commit id here, the api-version is not updated, so we don't need to update `assets.json`. Just trying to understand from the process of handling sync PR perspective, especially what shall we do if `api-version` is updated by updating commit id.